README.md

```diff
@@ -2,12 +2,19 @@
 
 Dejavu is a free, open source implementation of [Game Maker]. It is designed to be compatible with classic versions (8.0 and earlier) and eventually more recent versions (Game Maker: Studio). The goal is to preserve some of indie game dev history, and port it to modern platforms.
 
+You can **[try it out in your web browser]** right now!
+
+The current functionality is mainly focussed on GML support.
+
 [game maker]: https://en.wikipedia.org/wiki/GameMaker_Studio
+[try it out in your web browser]: https://dejavu.abubalay.com/
```

Comment:
```suggestion
You can [try it out][playground] in your web browser right now!

[game maker]: https://en.wikipedia.org/wiki/GameMaker_Studio
[playground]: https://dejavu.abubalay.com/
```

Reply:
Hmm, are you not keen on the bold? I'm fine with it either way, but I like drawing the user's eye to it.

Reply:
Yeah I could go either way, I mostly just wanted to decouple the link text from the markdown name for it. Feel free to re-bold if that looks better.